python_transport/wirepas_gateway/utils/argument_tools.py

```diff
@@ -241,7 +242,13 @@ def add_mqtt(self):
 
         self.mqtt.add_argument(
             "--mqtt_tls_version",
-            default=ssl.PROTOCOL_TLSv1_2,
+            default="PROTOCOL_TLSv1_2",
+            choices=["PROTOCOL_TLS",
```

Comment:
It looks like using PROTOCOL_TLSvx is deprecated: https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLSv1
So we should use PROTOCOL_TLS and then use OP_* flags.

But it is totally out of scope for this change and will be addressed in a different PR.

Thanks for your contribution!